cmd/empire/server.go

```diff
@@ -187,8 +187,10 @@ func newAuth(c *Context, e *empire.Empire) *auth.Auth {
 			},
 		}
 
+		org := c.String(FlagGithubOrg)
+
 		// After the user is authenticated, check their GitHub Organization membership.
-		if org := c.String(FlagGithubOrg); org != "" {
+		if org != "" {
```

Comment:
❔ Do I need to be checking if `org` is `nil` too? (Is that what `if org := <whatever>;` was checking while also simultaneously declaring and assigning to the variable? 🤯 )